hugo.toml

```diff
@@ -146,10 +146,20 @@ enable = false
 [params.links]
 # End user relevant links. These will show up on left side of footer and in the community page if you have one.
 [[params.links.user]]
-  name = "User mailing list"
+  name = "Post to Mailing List"
   url = "mailto:dev@parquet.apache.org"
   icon = "fa fa-envelope"
-  desc = "Discussion and help from your fellow users. To subscribe please email dev-subscribe@parquet.apache.org. View the archive here: https://lists.apache.org/list.html?dev@parquet.apache.org."
```

Comment:
I could not figure out how to make the content of this config file render as links. If someone can figure that we could potentially leave it as one icon/entry and just linkify these

I actually like a separate entry for the mailing list archives

Reply:
Hmm unfortunately we're kind of stuck with what docsy offers out of the box: https://github.com/google/docsy/blob/309da207e5deeda762348bfd0cc32367c5565a1e/layouts/partials/community_links.html#L29 which doesn't treat the `desc` or any of the content inside of it as link